index.js

```diff
@@ -11,3 +11,4 @@ module.exports.nlcconfig = require('./src/lib/nlcconfig');
 module.exports.nlcManager = require('./src/lib/nlcManager');
 module.exports.hubotPouch = require('./src/lib/PouchDB');
 module.exports.nlcDb = new DBManager({localDbName: 'nlc'});
```

Comment:
Let's use the database name from `env.db_nlc_local`.  You'll need to require `const env = require('./src/lib/env');`